### ISSUE TITLE
Add a pop method to scalar list update input

### ISSUE THREAD

**Issue by paulm17** [prisma/prisma#15464]
Right now we have the following functions: set, push, unset.

Would really like to see the pop ability.

Something like:

values:  [1, 2, 3, 4, 5]

```
data: {
   [field]: {
      pop: null
   }
}
```

Which would give:

values:  [1, 2, 3, 4]

Additionally, to be able to remove a specific item

```
data: {
   [field]: {
      pop: 2
   }
}
```

Which would give:

values:  [1, 3, 4, 5]

Finally, to be able to remove a specific items

```
data: {
   [field]: {
      pop: [2, 5]
   }
}
```

Which would give:

values:  [1, 3, 4]

Also sorry for the brevity, but hopefully the idea comes across!

Thanks for your consideration.  👍

**Comment by paulm17** [comment]
Also, add a starts_with and ends_with as well.

I have a nicely built dynamic query in javascript, to be converted into raw sql due to not having all this functionality.  😦

**Comment by FGoessler** [comment]
Hi,

We’ve noticed there hasn’t been activity on this issue for quite some time, and we suspect it might no longer be relevant. As part of our ongoing effort to streamline open issues in this repository, we’re closing this one to focus on what’s most important to our community.

If you believe this issue is still relevant and requires attention, feel free to reopen it. Reopening the issue is a valuable signal to us about what matters most to you.

You can learn more about our approach and motivations in the [Prisma ORM Manifesto](https://www.prisma.io/blog/prisma-orm-manifesto#1-defining-first-class-databases).

With love,
Your Prisma ORM Team ❤️

**Comment by paulm17** [comment]
Yes, this issue is still required and I've been patiently waiting for it.  @FGoessler please re-open it.
